pom.xml

```diff
@@ -17,9 +17,9 @@
   <description>This plugin allows users to create "folders" to organize jobs. Users can define custom taxonomies (like
     by project type, organization type etc). Folders are nestable and you can define views within folders. Maintained by CloudBees, Inc.
   </description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Folders+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/CloudBees+Folders+Plugin</url>
   <properties>
-    <jenkins.version>2.60.1</jenkins.version>
+    <jenkins.version>2.60.3</jenkins.version>
```

Comment:
`mvn hpi:run` testing of other plugins in #115 was awkward since actually the 2.60.x update center includes a bunch of plugins formally requiring 2.60.3. Best to use the last LTS in sequence.